textextraction/doc_process_toolkit.py

```diff
@@ -18,18 +18,23 @@ def get_doc_length(doc_text):
     return len(tuple(WORDS.finditer(doc_text)))
 
 
-def check_for_text(doc_path):
+def check_for_text(doc_path, extension):
```

Comment:
Wouldn't a simpler check be to try and extract the text from a document and see if that returns anything?